client/components/split-button/index.jsx

```diff
@@ -49,35 +49,17 @@ class SplitButton extends Component {
 		scary: false,
 	};
 
-	constructor() {
-		super( ...arguments );
-
-		this.state = {
-			isMenuVisible: false,
-			popoverContext: null,
-		};
-
-		this.showMenu = this.toggleMenu.bind( this, true );
-		this.hideMenu = this.toggleMenu.bind( this, false );
+	state = {
+		isMenuVisible: false,
+	};
 
-		this.setPopoverContext = this.setPopoverContext.bind( this );
-	}
+	popoverContext = React.createRef();
```

Comment:
TIL https://reactjs.org/docs/refs-and-the-dom.html#creating-refs :)

Reply:
me too!